localstack/testing/pytest/fixtures.py

```diff
@@ -1757,6 +1757,68 @@ def _create_rule(**kwargs):
         events_client.delete_rule(Name=rule["name"], EventBusName=rule["bus"])
 
 
+@pytest.fixture
+def ses_configuration_set(ses_client):
+    configuration_set_names = []
+
+    def factory(name: str) -> None:
+        ses_client.create_configuration_set(
+            ConfigurationSet={
+                "Name": name,
+            },
+        )
+        configuration_set_names.append(name)
+
+    yield factory
+
+    # this endpoint is not implemented for localstack
+    if os.environ.get("TEST_TARGET") == "AWS_CLOUD":
+        for configuration_set_name in configuration_set_names:
+            ses_client.delete_configuration_set(ConfigurationSetName=configuration_set_name)
+
+
+@pytest.fixture
+def ses_configuration_set_event_destination(ses_client):
```

Comment:
There seems to be 4 different destinations possible, maybe this should have `sns` in the name to be aware we would configure only SNS as an event destination?

It also seems SES needs to be allowed to publish to the SNS topic? (not currently enforced in LocalStack, but to be able to AWS-test it)
https://docs.aws.amazon.com/ses/latest/dg/event-publishing-add-event-destination-sns.html
Maybe we could add this policy to the topic while creating the configuration set event destination.

Reply:
That's weird because the test runs (mostly) unmodified on AWS. I only need to change the email addresses and remove the address verification.

Reply:
It seems topics get a default policy that allows the creator wide access:

```json
{
  "Version": "2008-10-17",
  "Id": "__default_policy_ID",
  "Statement": [
    {
      "Sid": "__default_statement_ID",
      "Effect": "Allow",
      "Principal": {
        "AWS": "*"
      },
      "Action": [
        "SNS:GetTopicAttributes",
        "SNS:SetTopicAttributes",
        "SNS:AddPermission",
        "SNS:RemovePermission",
        "SNS:DeleteTopic",
        "SNS:Subscribe",
        "SNS:ListSubscriptionsByTopic",
        "SNS:Publish"
      ],
      "Resource": "arn:aws:sns:eu-central-1:528541226335:test-topic-f5ba40b3",
      "Condition": {
        "StringEquals": {
          "AWS:SourceOwner": "528541226335"
        }
      }
    }
  ]
}
```

Because SES is in the same account, it gets access to `SNS:Publish`

Reply:
Ah, awesome, thanks for checking it!